sdk/cosmosdb/cosmos/test/public/integration/changeFeedIterator.spec.ts

```diff
@@ -211,6 +211,152 @@ describe("Change Feed Iterator", function (this: Suite) {
         },
       );
     });
+
+    it("startFromBeginning should fetch results for all partition keys if undefined partition key is provided", async function () {
```

Comment:
Take example of pk as ['key1', 'key2', 'key3'] : 
Add test case where partition key value is explicitly set as [undefined, undefined, undefined] ->it should fetch results where pk value matches undefined. 
[undefined, undefined] -> results for key1, key2 = undefined and all key3 values
[undefined]-> results for key1=undefined, all values for key2, key3

Reply:
This is handled in added test case 
it("should fetch correct results for undefined values of partition keys", async function () {